app/models/kpi.rb

```diff
@@ -0,0 +1,366 @@
+class Kpi < CouchRest::Model::Base
```

Comment:
Kpi model should not derive from CouchRest::Model::Base if it isn't being stored in the database.  From what I see, this model is used to pull back stuff from other models (Child, Incident, User).  If that is the case, this model doesn't need to be a CouchRest model.